server/app.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express')
+const routerTodo = require('./routes/todos.js')
```

Comment:
nanti ga cuman todo doang route nya , nambah userroutes juga 
jadi bikin 1 index js aja buat sisi routingan general
```suggestion
const routes = require('./routes')
```